flaml/tune/tune.py

```diff
@@ -344,8 +344,8 @@ def easy_objective(config):
                 # do cleanup operation here
                 return
     ```
-        search_alg: An instance of BlendSearch as the search algorithm
-            to be used. The same instance can be used for iterative tuning.
+        search_alg: An instance/string of the search algorithm
+            to be used. The same instance/string can be used for iterative tuning.

```

Comment:
```suggestion
            to be used. The same instance can be used for iterative tuning.
```
My understanding is that this sentence is about using the same instance for iterative tuning.

Reply:
resolved